arch/platform/native/Makefile.native

```diff
@@ -19,6 +19,10 @@ else
 CONTIKI_TARGET_SOURCEFILES += tun6-net.c
 endif
 
+ifeq ($(HOST_OS),Linux)
+TARGET_LIBFILES += -lrt
```

Comment:
Do we really need this? Travis seems to be passing even without?

Reply:
Yes, but only for glibc versions < 2.17 :)

Reply:
Aha! :^)

Reply:
Any particular reason for using an old glibc?

Reply:
Some OpenWRT targets are not using the most recent glibc if I remember correctly.

Reply:
Ah ok, fair enough!